tests/firebase_client_test.py

```diff
@@ -105,33 +132,54 @@ def testSetupAuthNoProjectId(self):
       with self.assertRaises(firebase_client.NoProjectIdError):
         self._client.SetupAuth()
 
-  @patch('firebase_admin.db.reference')
-  @patch('firebase_admin.initialize_app')
-  def testStart(self, mock_initialize_app, mock_db_ref):
+  def testStart(self):
     self._client.SetupAuth(project_id=TEST_PROJECT_ID)
     self._client.Start()
     self._client.subscription_complete.wait()
 
     debuggee_id = self._client._debuggee_id
 
-    mock_initialize_app.assert_called_with(
+    self._mock_initialize_app.assert_called_with(
         None, {'databaseURL': f'https://{TEST_PROJECT_ID}-cdbg.firebaseio.com'})
     self.assertEqual([
         call(f'cdbg/debuggees/{debuggee_id}'),
         call(f'cdbg/breakpoints/{debuggee_id}/active')
-    ], mock_db_ref.call_args_list)
+    ], self._mock_db_ref.call_args_list)
+
```

Comment:
A check of the writing of the Debuggee data to the debuggee_id ref should probably go in here as well?

Reply:
Good call.